share/completions/git.fish

```diff
@@ -798,6 +798,10 @@ format:\tSpecify which information to show"
     end
 end
 
+function __fish_git_is_rebasing
+    test -e (__fish_git rev-parse --git-path rebase-merge)
+end
```

Comment:
:thinking: It looks like `git -C path/to/other/repo --git-path rebase-merge` resolves to `.git/rebase-merge` instead of `path/to/other/repo/.git/rebase-merge`. So this is actually only hiding options like `git rebase --continue` if the *current* directory contains `.git/rebase-merge`, instead of the actual repo. So it might be best to just remove this check.

Reply:
Nevermind, I think I figured it out, but this check might still be overkill.